app/helpers/application_sidebar_helper.rb

```diff
@@ -20,7 +20,7 @@ def sidebar!
   #
   # @param [Array<String>] classes An array of classes to apply to the sidebar container.
   # @return [String] The buffer containing the markup for the sidebar.
-  def sidebar(classes: ['col-xs-7', 'col-sm-3', 'col-md-2'])
+  def sidebar(classes: ['col-lg-2', 'col-md-3', 'col-sm-4'])
```

Comment:
Was thinking of this, but when I tried viewing the page on `xs`, the sidebar takes up too much space. I'm proposing to collapse it into the navbar as seen in option 1 above, and because of that I didn't add the `col-xs-`